docs/posts/why-duckdb/index.qmd

```diff
@@ -0,0 +1,117 @@
+---
+title: "Why is DuckDB the default backend for Ibis?"
+author: "Phillip Cloud"
+date: "2024-02-16"
+categories:
+  - blog
+  - community
+---
+
+## Why DuckDB?
+
+Occasionally people ask us why DuckDB is the default backend.
+
+This blog posts aims to address this question.
+
+## A bit of history
+
+For years, Ibis had no default backend. We had somewhat hidden functionality to
+read parquet files with Pandas, but no one working on the project at the time
+was comfortable making Pandas execution the default.
+
+Pandas is great, but we didn't want to inherit all of its [technical
+baggage](https://wesmckinney.com/blog/apache-arrow-pandas-internals/) for
+users, especially new users, of Ibis.
+
+So, we decided not make any backend the official default because our primary
+local backend wouldn't give users a good out-of-the-box experience.
```

Comment:
```suggestion
So, we decided not make any backend the official default because our primary
option for local execution wouldn't give users a good out-of-the-box experience.
```

as an option - just to avoid saying 'backend' so many times

Reply:
Is it even a word anymore?